### DISCUSSION TITLE
项目无下载选项，出现两个警告node版本低和无法加载文件

### DISCUSSION THREAD

**Discussion by LoganQdf** [tw93/Pake#831] [Q&A]
<html>
<body>

The following actions use a deprecated Node.js version and will be forced to run on node20: actions/checkout@v3, actions/setup-node@v3, actions/cache/restore@v3, actions/upload-artifact@v3, actions/cache/save@v3. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/Show less
--
windows-latestNo files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded.

</body>
</html>

**Comment by LoganQdf** [comment]
主机上的node是22的

**Comment by Eufemlola** [comment]
我也是一样，生成的windows运行没窗口
